SerializationErrorCatch/src/main/java/cheetah/example/serializationerrorcatch/function/SerializationErrorCatchMapper.java

```diff
@@ -0,0 +1,23 @@
+package cheetah.example.serializationerrorcatch.function;
+
+import cheetah.example.serializationerrorcatch.model.InputEvent;
+import cheetah.example.serializationerrorcatch.model.OutputEvent;
+import org.apache.flink.api.common.functions.MapFunction;
+import org.apache.flink.api.common.functions.RichFilterFunction;
+import org.apache.flink.metrics.Gauge;
+
+/** SerializationErrorCatchMapper is a simple MapFunction that converts from InputEvent to OutputEvent. */
+public class SerializationErrorCatchMapper implements MapFunction<InputEvent, OutputEvent> {
+    private final String extraField;
+
+    public SerializationErrorCatchMapper(final String extraField) {
+        this.extraField = extraField;
+    }
+
+    @Override
+    public OutputEvent map(final InputEvent inputEvent) {
+        return new OutputEvent(inputEvent, extraField);
```

Comment:
any particular reason we dont just use the same type for input and output events?